src/client/app/components/ChartSelectComponent.tsx

```diff
@@ -40,32 +50,37 @@ export default class ChartSelectComponent extends React.Component<ChartSelectPro
 				<p style={labelStyle}>
 					<FormattedMessage id='graph.type' />:
```

Comment:
I'm not sure we need this label above the dropdown since it has wording inside. Maybe use the graph.type internationalized message to hold the "Select a graph type" below and replace with that.

Reply:
Please take a look at this: 

[https://imgur.com/a/3XpVE6e/all]

I've gone ahead and taken screenshots of what the dropdown menu would look like by itself and when users click on them with and without the label among other changes such as making the background white. Note the last two pictures are how the dropdown menu currently looks. We can proceed with any of these or I can try other suggestions as well.

Reply:
Thanks @LouisLu705. Having looked at all the options I think I like "dropdown w/ Outline and Graph Type label". This means I'm sticking with not using a background on the menu (before clicked) but changing so we keep the label. I think the consistency is valuable until we run low on vertical window space.

If you don't know, you can attach files/images with drop-n-drag to keep everything on the GitHub record.